app/src/main/java/com/github/sdp_begreen/begreen/social/GoogleAuth.kt

```diff
@@ -0,0 +1,45 @@
+package com.github.sdp_begreen.begreen.social
+
+import android.content.Context
+import android.content.Intent
+import com.github.sdp_begreen.begreen.R
+import com.google.android.gms.auth.api.signin.GoogleSignIn
+import com.google.android.gms.auth.api.signin.GoogleSignInClient
+import com.google.android.gms.auth.api.signin.GoogleSignInOptions
+
+
+
+object GoogleAuth {
+
+    // GoogleSignInClient object that can be used to initiate the Google sign-in process in an Android app
+    // This variable will hold the configured GoogleSignInClient object.
+    // Source : https://developers.google.com/android/reference/com/google/android/gms/auth/api/signin/GoogleSignInClient
```

Comment:
As Android Studio tells us, we should add `@SuppressLint("StaticFieldLeak")` above this variable to avoid memory leaks